Soundex.test.py

```diff
@@ -1,5 +1,6 @@
 import unittest
 from Soundex import generate_soundex
+from Soundex import encode_name
```

Comment:
Don't import encode_name in to test environment , because its not a part of public interface of Soundex module